ajax/libs/filesize/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "filesize",
+  "version": "3.5.6",
+  "filename": "filesize.min.js",
+  "description": "JavaScript library to generate a human readable String describing the file size",
+  "homepage": "https://filesizejs.com/",
+  "keywords": [
+    "file",
+    "filesize",
+    "size",
+    "readable",
+    "file system",
+    "bytes",
+    "diff"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/avoidwork/filesize.js"
+  },
+  "license": "BSD-3-Clause",
+  "author": "Jason Mulligan <@avoidwork> (https://avoidwork.com/)",
```

Comment:
The form of that doesn't reflect the actual `package.json`, it'd be `"author": "Jason Mulligan <jason.mulligan@avoidwork.com"`; the bower file content is probably questionable at this time (created & forgotten).